src/NuGet.Core/NuGet.Frameworks/def/IFrameworkNameProvider.cs

```diff
@@ -33,6 +33,11 @@ interface IFrameworkNameProvider
         /// </summary>
         bool TryGetShortProfile(string frameworkIdentifier, string profile, out string profileShortName);
 
+        /// <summary>
+        /// Get the official platform name from the short name.
+        /// </summary>
+        bool TryGetShortPlatform(string frameworkIdentifier, Version version, string platform, out string platformShortName);
```

Comment:
`platformVersion`?  My understanding is that it's only optional when `platform` is not specified (`string.Empty`). Although as I wrote in another comment, I don't understand why customers would use this API rather than `NuGetFramework.ToShortFolderName()`.